### PR TITLE
Updated and made project Android Studio compatible

### PR DESCRIPTION
**Pull request by JudeRosario**
1. Added a `build.gradle` file
2. Updated `AndroidManifest.xml` for Lollipop Compatibility
3. Updated Readme.md with instructions for AS
4. Tested and used code up to Lollipop (API 21)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apply plugin: 'com.android.library'
+
+dependencies {
+    compile 'com.android.support:support-v4:21.0.3'
```
Comment (jayschwa) [inline build.gradle]: Is this necessary? I thought the support libraries were only needed for backporting features to older Android versions, and I can't think of anything offhand in this project that would need it.

Comment (JudeRosario) [inline build.gradle]: Agreed, I am so used to these support libraries that I instinctively include them. Fixed in the latest commit :)

**Comment by jayschwa** [comment]
Thanks!

**Comment by JudeRosario** [comment]
Thank you for sharing this :+1:

**Pull request merged**